core/ResManager/plResManager.cpp

```diff
@@ -156,33 +156,42 @@ plPageInfo* plResManager::ReadPage(const ST::string& filename, bool stub) {
         }
     }
 
-    hsFileStream* S = new hsFileStream();
-    if (!S->open(file, fmRead))
-        throw hsFileReadException(__FILE__, __LINE__, file.c_str());
+    if (packed) {
+        hsFileStream prpS;
+        if (!prpS.open(file, fmRead))
+            throw hsFileReadException(__FILE__, __LINE__, file.c_str());
+        // hsFileStream destructor closes the file
+        return ReadPage(&prpS, &prpS, stub);
+    } else {
```

Comment:
Does anyone actually use this code path?  We could probably get rid of it to simplify things a bit...

Reply:
@dpogue

Reply:
Realistically, nobody is going to use this code, but it allows for some fun experimenting so I'd prefer to keep it if it's not a maintenance burden (which it doesn't seem to be?)